ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -210,6 +201,37 @@ def _check_enabled_features(self):
         attributes = parsed["data"]["attributes"]
         return attributes["code_coverage"], attributes["tests_skipping"]
 
+    def _configure_itr(self, api_key, app_key, requests_mode):
```

Comment:
I have mixed feelings about these parameters vs using `self` , but I usually don't like to rely on order of operations when `__init__()` calls other methods that use `self` (although I can't really explain the exception I made for `self._code_coverage_enabled_by_api` below).

Reply:
Right... I need to drop this habit of commenting on my own PRs for clarification and then blocking myself with unresolved conversations. 😅